instagram/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import dummyData from '../../dummy-data';
+import PostContainer from '../PostContainer/PostContainer';
+import SearchBar from '../SearchBar/SearchBar';
+
+class PostsPage extends Component {
+  constructor() {
+    super();
+    this.state = {
+      postData: [],
+      search: ''
+    };
+  }
+
+  componentDidMount() {
+    setTimeout(() => {
+      this.setState({
+        postData: dummyData
+      });
+    }, 800);
+  }
+
+  search = event => {
+    this.setState({
+      search: event.target.value.substr(0, 20)
```

Comment:
why limit the search to 20 characters? ... that's not how the real instagram handles searches and it generates imo a bad user experience.